_usage/command-line.md

```diff
@@ -1,212 +1,121 @@
 ---
 layout: default
-title: Using the command line
+title: Connecting to Babelfish
 nav_order: 1
 ---
 
-## Using client tools to work with Babelfish
+## Using client tools to connect to Babelfish
 
-Most people use SQL Server Management Studio (SSMS) on Microsoft Windows to
-connect to Babelfish.
-
-However, there is an ever-growing community which prefers to use the command
-line to work with the server.
-Those users have the following command-line tool choices on Linux:
+On Windows, most people use SQL Server Management Studio (SSMS) or `sqlcmd` to
+connect to SQL Server, and these can be used to connect to Babelfish as well.
+SSMS is not available on Linux, so you are limited to using `sqlcmd` there.
 
-- sqlcmd: Using the Microsoft SQL Server interface
-- tsql: Using the FreeTDS command line tool
+Other clients that speak the TDS protocol can also be used, but are not
+officially supported by Babelfish.  On Linux, these include the FreeTDS command
+line tool `tsql`.
 
-Note that there is also a `psql` version for Babelfish, but that is not reliable.
-Therefore, it is strongly recommended not to use it.
+Using the PostgreSQL protocol, you can use `psql` or any other client to
```

Comment:
How about: 

You can use a PostgreSQL client to connect to Babelfish on the PostgreSQL port. psql and pgAdmin are examples of open-source clients that run on Windows and Linux platforms, that use native PostgreSQL dialect.

Reply:
Done.